lib/reporting.js

```diff
@@ -18,11 +18,17 @@ function payload(attribute, min, max, change, overrides) {
     return [payload];
 }
 
-async function readEletricalMeasurementMultiplierDivisors(endpoint) {
+// Fix the problem that commit #3839 introduced.
+// You can set readFrequencyAttrs = true if the device support acFrequencyDivisor and acFrequencyMultiplier
+// See Develco.js SPLZB-132 for example
+async function readEletricalMeasurementMultiplierDivisors(endpoint, readFrequencyAttrs = false) {
```

Comment:
This will disable acFrequencyDivisor/Multiplier read for all devices except when explicitly enabled. I would opt for the reverse, by default it is read so `readFrequencyAttrs = false` -> `readFrequencyAttrs = true`

Reply:
Won't that require readFrequencyAttrs = false for all other devices that support the readEletricalMeasurementMultiplierDivisors but don't support acFrequencyDivisor and acFrequencyMultiplier ??

@corneyl found that only 2 devices so far supported the frequency attribute.

Personally I think it's quite a change for "only" 2 devices so far. I mean they probably work even if the attribute is disabled, but the other way around, almost nothing works on all the other devices

Reply:
@Koenkk As far as I can see there are about 19 devices that call `readEletricalMeasurementMultiplierDivisors()` of which only one actually uses the frequency attributes, which is the Develco SPLZB-132 device. So then reading them not by default would be a more logical choice, right?

Also, I'm afraid more than just this Heimann plug have now problems reporting the power usage, see e.g. 
- https://github.com/Koenkk/zigbee2mqtt/issues/11983
- https://github.com/Koenkk/zigbee2mqtt/issues/11943

These issues show the same UNSUPPORTED_ATTRIBUTE error in the logs.

Reply:
You are right guys, only `SPLZB-132` sets up the ac frequency reporting. Thanks!